.travis.yml

```diff
@@ -1,16 +1,15 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.2
+  - 2.6.4
 gemfile:
   - gemfiles/rails4.2.gemfile
   - gemfiles/rails5.0.gemfile
   - gemfiles/rails5.1.gemfile
```

Comment:
I think that we can remove `4.2`, `5.0`, and `5.1` to cut the test running time and because they're no longer supported.